fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateToExternalTableScanRule.java

```diff
@@ -45,6 +45,8 @@ public class PushDownPredicateToExternalTableScanRule extends TransformationRule
             new PushDownPredicateToExternalTableScanRule(OperatorType.LOGICAL_MYSQL_SCAN);
     public static final PushDownPredicateToExternalTableScanRule JDBC_SCAN =
             new PushDownPredicateToExternalTableScanRule(OperatorType.LOGICAL_JDBC_SCAN);
+//    public static final PushDownPredicateToExternalTableScanRule PIXELS_SCAN =
```

Comment:
Add a line of comment for this line if it is the switch to enable predicate pushdown.